### PR TITLE
main/collectd: upgrade to collectd 5.8.1

### PR DESCRIPTION
**Pull request by ad-m**

### CODE REVIEW

File: main/collectd/APKBUILD [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Contributor: Łukasz Jendrysik <scadu@yandex.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=collectd
-pkgver=5.8.0
-pkgrel=4
+pkgver=5.8.1
+pkgrel=0
 pkgdesc="The system statistics collection daemon"
 url="http://collectd.org"
 arch="all"
```
Comment (maxice8) [inline main/collectd/APKBUILD]: GPL-2.0 is not a valid SPDX identifier, aports will detect that when spdx-licenses-list

File: main/collectd/APKBUILD [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Contributor: Łukasz Jendrysik <scadu@yandex.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=collectd
-pkgver=5.8.0
-pkgrel=4
+pkgver=5.8.1
+pkgrel=0
 pkgdesc="The system statistics collection daemon"
 url="http://collectd.org"
```
Comment (maxice8) [inline main/collectd/APKBUILD]: url= is now https://

**Comment by ad-m** [comment]
> Empty variables can be removed for brevity (depends=).

Can we do it globally?

**Comment by maxice8** [comment]
They are being done as the APKBUILD is touched

**Comment by ad-m** [comment]
Fixed.

**Pull request closed without merging**